src/components/App/AppLoader.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import LoadingScreen from '../LoadingScreen/LoadingScreen'
+
+const INTERNAL_APPS = ['home', 'permissions', 'apps', 'settings']
+
+const LOADING_ORG = 'Loading organization…'
+const LOADING_APPS = 'Loading apps…'
+const LOADING_APP = 'Loading {APP}…'
```

Comment:
```JavaScript?
const LOADING_APP = app => `Loading ${app}`
```
To avoid the replace later on?

Reply:
I would like to avoid mixing two different types… do you think it would be preferable to have them all as functions?